samples/rtl/fladd.py

```diff
@@ -0,0 +1,241 @@
+import dace
+from dace.transformation.dataflow import StreamingMemory
+from dace.transformation.interstate import FPGATransformState
+from dace.transformation.dataflow import TrivialMapElimination
+import numpy as np
+
+# add symbol
+N = dace.symbol('N')
+
+# add sdfg
+sdfg = dace.SDFG('fladd')
+
+# add state
+state = sdfg.add_state('device_state')
+
+# add parameter
+veclen = 1
+sdfg.add_constant('VECLEN', veclen)
+sdfg.add_constant('DATA_WIDTH', 256)
+sdfg.add_constant('WORD_WIDTH', 32)
+sdfg.add_constant('RATIO', 2)
+
+# add arrays
+sdfg.add_array('A', [N // veclen],
+               dtype=dace.vector(dace.float32, veclen),
+               storage=dace.StorageType.CPU_Heap)
+sdfg.add_array('B', [N // veclen],
+               dtype=dace.vector(dace.float32, veclen),
+               storage=dace.StorageType.CPU_Heap)
+sdfg.add_array('C', [N // veclen],
+               dtype=dace.vector(dace.float32, veclen),
+               storage=dace.StorageType.CPU_Heap)
+sdfg.add_array('fpga_A', [N // veclen],
+               dtype=dace.vector(dace.float32, veclen),
+               transient=True,
+               storage=dace.StorageType.FPGA_Global)
+sdfg.add_array('fpga_B', [N // veclen],
+               dtype=dace.vector(dace.float32, veclen),
+               transient=True,
+               storage=dace.StorageType.FPGA_Global)
+sdfg.add_array('fpga_C', [N // veclen],
+               dtype=dace.vector(dace.float32, veclen),
+               transient=True,
+               storage=dace.StorageType.FPGA_Global)
+
+# add streams
+sdfg.add_stream('A_stream',
+                buffer_size=32,
+                dtype=dace.vector(dace.float32, veclen),
+                transient=True,
+                storage=dace.StorageType.FPGA_Local)
+sdfg.add_stream('B_stream',
+                buffer_size=32,
+                dtype=dace.vector(dace.float32, veclen),
+                transient=True,
+                storage=dace.StorageType.FPGA_Local)
+sdfg.add_stream('C_stream',
+                buffer_size=32,
+                dtype=dace.vector(dace.float32, veclen),
+                transient=True,
+                storage=dace.StorageType.FPGA_Local)
+
+# add custom rtl tasklet
+rtl_tasklet = state.add_tasklet(name='rtl_tasklet',
+                                inputs={'a', 'b'},
+                                outputs={'c'},
+                                code='''
+    /*
+        This tasklet tests whether a contineous stream of data can be processed
```

Comment:
```suggestion
        This tasklet tests whether a continuous stream of data can be processed
```

Reply:
Fixed in https://github.com/carljohnsen/dace/commit/11984d3c2c997860d2124ef82133077ccd84e753